.gitlab-ci.yml

```diff
@@ -153,12 +153,12 @@ spellcheck:
     stage:                           workspace
     <<:                              *docker-env
     script:
-        - cargo spellcheck check --cfg=.config/cargo_spellcheck.toml --checkers hunspell --code 1
+        - cargo spellcheck check -vvv --cfg=.config/cargo_spellcheck.toml --checkers hunspell --code 1
```

Comment:
It's much much fast if you can pass all files at once, if that does not make the stack explode.

Most time is used to load the builtin rules, if you can make that a onetime offset, it could safe you a couple of `100ms x #files`